### PR TITLE
Get all tasks for caseworker by caseworker Id

### PR DESCRIPTION
**Pull request by DommarajuUmamaheswari**

### CODE REVIEW

File: src/Kmd.Momentum.Mea/TaskApi/Model/AssignedActors.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Momentum.Mea.TaskApi.Model
+{
+    public class AssignedActors
+    {
+        public string Id { get; set; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/AssignedActors.cs:9]: is this id string? isin't it guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/Participants.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Momentum.Mea.TaskApi.Model
+{
+    public class Participants
+    {
+        public string ActorId { get; set; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/Participants.cs:9]: string or Guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/Reference.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Kmd.Momentum.Mea.TaskApi.Model
+{
+    public class Reference
+    {
+        public string Id { get; set; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/Reference.cs:5]: string or Guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+
+namespace Kmd.Momentum.Mea.TaskApi.Model
+{
+    public class TaskData
+    {
+        public string SicknessBenefitType { get; set; }
+        public string Id { get; set; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs:8]: string or Guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+        public string UpdatedById { get; set; }
+
+        public string CreatedById { get; set; }
+        public string StateChangedAt { get; set; }
+        public IReadOnlyList<string> Assignee { get; set; }
+        public IReadOnlyList<AssignedActors> AssignedActors { get; set; }
+        public string Subject { get; set; }
+        public int Type { get; set; }
+        public string RelatedEntityId { get; set; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs:29]: string or Guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs [hunk 6]
```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace Kmd.Momentum.Mea.TaskApi.Model
+{
+    public class TaskDataResponseModel
+    {
+        public string TaskId { get; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs:7]: string or Guid?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+        public string Title { get; }
+
+        public string Description { get; }
+
+        public string Deadline { get; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs:13]: string or datetime?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+        public string CreateAt { get; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs:15]: string or datetime?

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+        public string StateChangedAt { get; }
```
Comment (amitxagarwal) [inline src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs:17]: string or datetime?

File: src/Kmd.Momentum.Mea/TaskApi/Model/AssignedActors.cs (hunk 1 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/AssignedActors.cs]: changed to guid

File: src/Kmd.Momentum.Mea/TaskApi/Model/Participants.cs (hunk 2 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/Participants.cs]: changed to guid

File: src/Kmd.Momentum.Mea/TaskApi/Model/Reference.cs (hunk 3 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/Reference.cs:5]: This is string in momemtum Api

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs (hunk 4 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs]: changed to guid

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs (hunk 5 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskData.cs]: changed to guid

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs (hunk 6 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs]: changed to guid

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs (hunk 7 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs]: changed to dateTime

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs (hunk 8 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs]: changed to datetime

File: src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs (hunk 9 above)
Comment (DommarajuUmamaheswari) [reply src/Kmd.Momentum.Mea/TaskApi/Model/TaskDataResponseModel.cs]: changed to datetime

**Comment by DommarajuUmamaheswari** [comment]
> Did you checked this with MCA contract whether these ID's are actually guid?

Checked with MCA contract

File: test/Kmd.Momentum.Mea.Integration.Tests/Caseworker/CaseworkerTests.cs [hunk 10]
```diff
@@ -109,5 +110,51 @@ public async Task GetCaseworkerByCaseworkerIdFails()
...
+        [SkipLocalFact]
+        public async Task GetAllTasksByCaseworkerIdSuccess()
+        {
+            //Arrange       
+            var client = _factory.CreateClient();
+            var caseworkerId = "836e2ad4-d028-4e3d-bb01-fdd60bca9b81";
+            var requestUri = $"/caseworkers/{caseworkerId}/tasks?pageNumber=1";
+            var tokenHelper = new TokenGenerator();
+            var accessToken = await tokenHelper.GetToken().ConfigureAwait(false);
+
```
Comment (amitxagarwal) [inline test/Kmd.Momentum.Mea.Integration.Tests/Caseworker/CaseworkerTests.cs:123]: there is a common method to get the token from token generator. Why don't you use that?

**Pull request merged**